### PR TITLE
Extended docs for new assets addition and system clock not synced

### PR DESCRIPTION
**Pull request by vnavascues**
- Extended "adding new assets" section with the steps to follow for getting the CoingGecko/CryptoCompare asset ID (e.g. try the API, run the specific test, etc. as per https://github.com/rotki/rotki/pull/1826/files#r532199748).
- Added a general Troubleshooting section in users guide.
- Fixed styles I broke in previous releases.

### CODE REVIEW

**Comment by LefterisJP** [comment]
@vnavascues This should go to develop. Not bugfixes. The only thing going to bugfixes, is bugfixes and other stuff that are supposed to be included in a potential patch release.

File: docs/contribute.rst [hunk 1]
```diff
@@ -73,31 +75,53 @@ To add new assets to rotki you need to edit `all assets file <https://github.com
...
 - ``forked``: This is an optional field. Given to specify if an asset is a fork of another asset. For example ``ETC`` should have ``ETH`` here.
 - ``swapped_for``: This is an optional field. Given to specify if an asset is swapped for another asset. For example ``LEND`` should have ``AAVE`` here.
 
-Once an asset is added here the md5sum of the file should be regenerated and added to the `meta file <https://github.com/rotki/rotki/blob/239552b843cd8ad99d02855ff95393d6032dbc57/rotkehlchen/data/all_assets.meta>`__. And the version in the meta file should also be bumped. The same changes should be done in the unit test that checks exactly for the md5 sum of the assets file.
+Once an asset is added and both CoinGecko and CryptoCompare identifiers have been validated (if exist), the md5sum of the file should be regenerated and added to the `meta file <https://github.com/rotki/rotki/blob/239552b843cd8ad99d02855ff95393d6032dbc57/rotkehlchen/data/all_assets.meta>`__. And the version in the meta file should also be bumped. The same changes should be done in the unit test that checks exactly for the md5 sum of the assets file.
```
Comment (LefterisJP) [inline docs/contribute.rst:89]: if exist -> if existing

**Comment by vnavascues** [comment]
It seems I take docs as important as code :D. Ok, I will close it and open it against develop.

**Comment by LefterisJP** [comment]
Apart from the typo, looks good, but reopen the PR and make against develop. Can't find a way to retarget an existing PR. Perhaps there is one.

**Comment by LefterisJP** [comment]
Ah wait wait no.

**Comment by LefterisJP** [comment]
It's actually doable. I pressed "Edit" and changed the base branch. And sicne there are no conflicts it was easy.

File: docs/contribute.rst [hunk 2]
```diff
@@ -73,31 +75,53 @@ To add new assets to rotki you need to edit `all assets file <https://github.com
...
 - ``forked``: This is an optional field. Given to specify if an asset is a fork of another asset. For example ``ETC`` should have ``ETH`` here.
 - ``swapped_for``: This is an optional field. Given to specify if an asset is swapped for another asset. For example ``LEND`` should have ``AAVE`` here.
 
-Once an asset is added here the md5sum of the file should be regenerated and added to the `meta file <https://github.com/rotki/rotki/blob/239552b843cd8ad99d02855ff95393d6032dbc57/rotkehlchen/data/all_assets.meta>`__. And the version in the meta file should also be bumped. The same changes should be done in the unit test that checks exactly for the md5 sum of the assets file.
+Once an asset is added and both CoinGecko and CryptoCompare identifiers have been validated (if existing), the md5sum of the file should be regenerated and added to the `meta file <https://github.com/rotki/rotki/blob/239552b843cd8ad99d02855ff95393d6032dbc57/rotkehlchen/data/all_assets.meta>`__. And the version in the meta file should also be bumped. The same changes should be done in the unit test that checks exactly for the md5 sum of the assets file.
```
Comment (vnavascues) [inline docs/contribute.rst:89]: @LefterisJP amended

**Review by LefterisJP — APPROVED** [review]
looking good, thanks

**Comment by LefterisJP** [comment]
test failure unrelated timeout. Will increase it myself. Merging.

**Pull request merged**